src/uct/base/uct_mem.c

```diff
@@ -56,14 +56,15 @@ static inline int uct_mem_get_mmap_flags(unsigned uct_mmap_flags)
     return mm_flags;
 }
 
-ucs_status_t uct_mem_alloc(void *addr, size_t min_length, unsigned flags,
-                           uct_alloc_method_t *methods, unsigned num_methods,
-                           uct_md_h *mds, unsigned num_mds,
-                           const char *alloc_name, uct_allocated_memory_t *mem)
+ucs_status_t uct_mem_alloc(const uct_mem_alloc_params_t *param,
```

Comment:
param -> params

Reply:
Done. Also changed param->params elsewhere for consistency.